install-worker.sh

```diff
@@ -44,7 +45,7 @@ sudo systemctl enable iptables-restore
 ################################################################################
 
 sudo yum install -y yum-utils device-mapper-persistent-data lvm2
-sudo yum install -y docker
+sudo amazon-linux-extras install -y docker
```

Comment:
👍 
In the interest of being explicit, I might write this as two separate lines:
```
sudo amazon-linux-extras enable docker
sudo yum install -y docker
```

Reply:
I make the changes as your comment, and fully agree is a lot more clear that way